lib/networks.js

```diff
@@ -103,7 +105,17 @@ addNetwork({
     'seed.bitcoinstats.com',
     'seed.bitnodes.io',
     'bitseed.xf2.org'
-  ]
+  ],
+  genesis: new Buffer('010000000000000000000000000000000000000000000000000000000000' +
```

Comment:
This is the whole block, but for testnet you used only the hash. Please pick one (I'd pick the hash)